pkg/server/authserver/authentication_test.go

```diff
@@ -119,7 +133,7 @@ func TestSSLEnforcement(t *testing.T) {
 	insecureContext := newRPCContext(true, username.TestUserName())
 
 	kvGet := &kvpb.GetRequest{}
-	kvGet.Key = roachpb.Key("/")
```

Comment:
nit: this seems like not an equivalent change - why are we not appending `"/"` to the tenant prefix?